### ISSUE TITLE
The "Query Tor via its API" Python Scripts Not Working - Ubuntu 22.04

### ISSUE THREAD

**Issue by war59312** [toralf/torutils#11]
Hi,

First off, thanks a ton for all the hard work and sharing.

Sadly, not able to get the "Query Tor via its API" Python Scripts to work on my Ubuntu 22.04 VM.

info.py:

```
sudo python3 info.py --address 127.0.0.1 --ctrlport 9051
 0.4.7.10   uptime: 15:55   flags: none
Traceback (most recent call last):
  File "/home/will/info.py", line 210, in <module>
    main()
  File "/home/will/info.py", line 84, in main
    relays = parse_consensus(
  File "/home/will/info.py", line 40, in parse_consensus
    for desc in parse_file(filename):
  File "/usr/local/lib/python3.10/dist-packages/stem/descriptor/__init__.py", line 375, in parse_file
    for desc in handler(descriptor_file, descriptor_type, validate, document_handler, **kwargs):
  File "/usr/local/lib/python3.10/dist-packages/stem/descriptor/__init__.py", line 450, in _parse_file_for_path
    with open(descriptor_file, 'rb') as desc_file:
FileNotFoundError: [Errno 2] No such file or directory: '/var/lib/tor/data/cached-consensus'
```

ps.py:

```
sudo python3 ps.py  --address 127.0.0.1 --ctrlport 9051
Traceback (most recent call last):
  File "/home/will/ps.py", line 205, in <module>
    main()
  File "/home/will/ps.py", line 66, in main
    relays = parse_consensus(
  File "/home/will/ps.py", line 29, in parse_consensus
    for desc in parse_file(filename):
  File "/usr/local/lib/python3.10/dist-packages/stem/descriptor/__init__.py", line 375, in parse_file
    for desc in handler(descriptor_file, descriptor_type, validate, document_handler, **kwargs):
  File "/usr/local/lib/python3.10/dist-packages/stem/descriptor/__init__.py", line 450, in _parse_file_for_path
    with open(descriptor_file, 'rb') as desc_file:
FileNotFoundError: [Errno 2] No such file or directory: '/var/lib/tor/data/cached-consensus'
```

Tor version 0.4.7.10 was installed at the normal default location and has been working fine for over a year now.

Thanks for assisting.

Best,

Will

**Comment by toralf** [comment]
I pushed a fix for Debian - pls test it : 037858d

**Comment by war59312** [comment]
Perfect, TY.

Success 🥇 

 ```
sudo python3 info.py --address 127.0.0.1 --ctrlport 9051

 0.4.7.10   uptime: 17:18:16   flags: none
+------------------------------+-------+-------+
| Type                         |  IPv4 |  IPv6 |
+------------------------------+-------+-------+
| Inbound to OR from relay     |       |       |
| Inbound to OR from non-relay |       |       |
| Inbound to ControlPort       |       |       |
| Outbound to relay OR         |    93 |       |
| Outbound to relay non-OR     |       |       |
| Outbound as Exit             |       |       |
| Outbound unknown             |    14 |       |
+------------------------------+-------+-------+
| Total                        |   107 |       |
+------------------------------+-------+-------+
 relay OR connections               93       0
          unique ips                93       0
```
